src/sage/calculus/calculus.py

```diff
@@ -1203,7 +1214,7 @@ def limit(ex, dir=None, taylor=False, algorithm='maxima', **argv):
 
         sage: var('a')
         a
-        sage: limit(x^a,x=0)
+        sage: limit(x^a,x=0)  # random - maxima 5.46.0 does not need extra assumption
```

Comment:
```suggestion
        sage: limit(x^a,x=0)
```

Reply:
oops, right, I miscounted the number of assumptions it still needs. Good catch.